templates/userdata.tmpl

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+/usr/local/bin/aws configure set profile.default.region eu-west-1
+/usr/local/bin/aws configure set profile.default.output json
+/usr/local/bin/aws configure set profile.ebs.role_arn $(wget -q -O - 169.254.169.254/latest/meta-data/iam/info | grep "InstanceProfileArn" | awk -F\"  '$2 == "InstanceProfileArn" {print $4}')
```

Comment:
Why not incoporate the grep into the awk command? Not a problem, just a thought.
Also isn't curl a lighterweight option than wget for this type of thing?